### PR TITLE
Layouts - set typeId in layout level

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Description
add required `typeId` field in `layout`

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Comment by kirbles19** [comment]
@Itay4 Updated.

File: Packs/CommonTypes/ReleaseNotes/1_8_7.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Layouts
+##### layout-edit-Vulnerability.json
+Set default incident type.
+##### layout-details-Vulnerability.json
+Set default incident type.
```
Comment (yaakovi) [inline Packs/CommonTypes/ReleaseNotes/1_8_7.md:6]:
```suggestion
Set the default incident type for the layout.
##### layout-details-Vulnerability.json
Set the default incident type for the layout.
```

File: Packs/Phishing/ReleaseNotes/1_10_1.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Layouts
+##### layout-quickView-Phishing.json
+Set default incident type.
+##### layout-mobile-Phishing.json
+Set default incident type.
+##### layout-edit-Phishing.json
+Set default incident type.
```
Comment (yaakovi) [inline Packs/Phishing/ReleaseNotes/1_10_1.md:8]:
```suggestion
Set the default incident type for the layout.
##### layout-mobile-Phishing.json
Set the default incident type for the layout.
##### layout-edit-Phishing.json
Set the default incident type for the layout.
```

**Review by yaakovi — APPROVED** [review]
@kirbles19 missed a few

**Pull request merged**